ci/build-helpers/publishing/build.gradle.kts

```diff
@@ -26,7 +26,9 @@ dependencies {
     val jacksonVersion = "2.12.5"
     implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-xml:$jacksonVersion")
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion")
-    implementation("io.ktor:ktor-client-okhttp:2.3.13")
+    implementation("io.ktor:ktor-client-core:3.1.3")
+    implementation("io.ktor:ktor-client-cio:3.1.3")
+    implementation("io.ktor:ktor-client-okhttp:3.1.3")
```

Comment:
I beleave we need only one

Reply:
We use CIO for the new Maven Central API, OkHttp is used for Maven Space API, which is better to keep as is